platform/pwl.c

```diff
@@ -816,68 +824,68 @@ window_dispatch_key_event (PwlWindow  *self,
 
 
 static gboolean
-display_on_keyboard_repeat_timeout (PwlDisplay *self)
+display_on_keyboard_repeat_timeout (PwlSeat *pwl_seat)
```

Comment:
This function acts on a `PwlSeat` so I would rename it as `seat_on_keyboard_repeat_timeout` and continue calling the first parameter `self`, which makes the refactor do less changes:

```c
static gboolean
seat_on_kayboard_repeat_timeout (PwlSeat *self)
{
    /* No need to rename s/self/seat/ inside the function. */
}
```

Reply:
done.